<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>management_signup</class>
 <widget class="QDialog" name="management_signup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>60</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Employee No.:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="staff_number">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>70</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Please enter a number</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>70</y>
     <width>60</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Name: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>70</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Please enter your name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>60</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Gender: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="gender">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>120</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Please enter your gender</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>120</y>
     <width>60</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>ID No.: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="id_number">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>120</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Please enter your ID No.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>170</y>
     <width>60</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Phone No.: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="phone">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>170</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Please enter your phone no.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>170</y>
     <width>60</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Email: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="email">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>170</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Please enter your Email</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>390</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>390</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>220</y>
     <width>60</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Department</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>220</y>
     <width>60</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Dep. No.: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>270</y>
     <width>60</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Position: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>270</y>
     <width>60</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Username: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>320</y>
     <width>60</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Password: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>320</y>
     <width>60</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm Password: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="department">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>220</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Please enter your department name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="position">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>270</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Please enter your position</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="password">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>320</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Customize</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="department_number">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>220</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Please enter a number</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="account">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>270</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Customize</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_12">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>320</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Same as password</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>management_signup</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>482</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>484</x>
     <y>308</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
